Core/include/Acts/Propagator/StraightLineStepper.hpp

```diff
@@ -344,13 +344,12 @@ class StraightLineStepper {
       // Extend the calculation by the time propagation
       // Evaluate dt/dlambda
       D(3, 7) = h * state.options.mass * state.options.mass *
-                (state.stepping.q == 0. ? 1. : state.stepping.q) /
-                (state.stepping.p * dtds);
+                (state.stepping.q == 0. ? 1. : state.stepping.q) / (p * dtds);
```

Comment:
```suggestion
                state.stepping.pars[eFreeQOverP] / dtds;
```
That should be the same, right?

Reply:
Right. Fixed.